.github/workflows/check.yml

```
@@ -26,23 +26,18 @@ jobs:
           fetch-depth: 50
           
       - name: Setup | Rust
-        uses: actions-rs/toolchain@v1
+        uses: dtolnay/rust-toolchain@stable
         with:
-          toolchain: nightly
-          profile: minimal
-          target: ${{ matrix.target }}
+          toolchain: stable
+          targets: ${{ matrix.target }}
 
       - name: Setup DVC
         uses: iterative/setup-dvc@v1
         
       - name: Rust Cache
         uses: Swatinem/rust-cache@v2
-
-      - uses: actions-rs/cargo@v1
-        with:
-          use-cross: true
-          args: --release --target=${{ matrix.target }} --features native-tls/vendored --locked
-          command: build
+      - name: Build target
+        run: cargo build --release --target=${{ matrix.target }} --features native-tls/vendored --locked
```

The intent would be to run `build` using `cross` 

```suggestion
      - name: Build target using cross
        run: cross build --release --target=${{ matrix.target }} --features native-tls/vendored --locked
```